.github/workflows/deploy-base.yaml

```diff
@@ -8,7 +8,7 @@
 on:
   workflow_call:
     inputs:
-      env-name:
+      environment:
```

Comment:
rename this input, with cascading changes in the deploy workflows that call `deploy-base.yaml`